web/sites/default/config/field.field.node.park_facility.panelizer.yml

```diff
@@ -0,0 +1,26 @@
+uuid: fcfe1afe-246b-49a8-9dcb-52597fd9641a
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.panelizer
+    - node.type.park_facility
+  module:
+    - custom_add_another
+    - panelizer
+third_party_settings:
+  custom_add_another:
+    custom_add_another: ''
+    custom_remove: ''
+id: node.park_facility.panelizer
+field_name: panelizer
+entity_type: node
+bundle: park_facility
+label: Panelizer
+description: ''
+required: false
+translatable: true
+default_value: {  }
+default_value_callback: ''
+settings: {  }
+field_type: panelizer
```

Comment:
Was panelizer a forced requirement here? We are not using it anywhere else. I was hoping we would be able to remove it from our enabled modules at some point.